customize_image.sh

```diff
@@ -327,9 +327,10 @@ if [ "$MINIMG" != "1" ]; then
 		--output "$IMAGEDIR/$NODEREDSCRIPT"
 	chmod 755 "$IMAGEDIR/$NODEREDSCRIPT"
 	NODERED_VER="3.0.2"
+	REVPI_NODES_VERSION="1.1.0"
 	chroot "$IMAGEDIR" /usr/bin/sudo -u pi $NODEREDSCRIPT --confirm-install --confirm-pi --no-init --nodered-version="$NODERED_VER"
 	rm "$IMAGEDIR/$NODEREDSCRIPT"
-	chroot "$IMAGEDIR" /usr/bin/sudo -u pi /usr/bin/npm install --prefix /home/pi/.node-red node-red-contrib-revpi-nodes
+	chroot "$IMAGEDIR" /usr/bin/sudo -u pi /usr/bin/npm install --prefix /home/pi/.node-red "git://github.com/RevolutionPi/node-red-contrib-revpi-nodes#${REVPI_NODES_VERSION}"
```

Comment:
This requires git to be installed and available to the npm command as this will clone the whole repository.

You could use the tarball of an release instead:
```suggestion
	chroot "$IMAGEDIR" /usr/bin/sudo -u pi /usr/bin/npm install --prefix /home/pi/.node-red "https://github.com/RevolutionPi/node-red-contrib-revpi-nodes/archive/refs/tags/${REVPI_NODES_VERSION}.tar.gz"
```

Reply:
Great idea. I've updated the PR